### PR TITLE
Homework 6

### PR DESCRIPTION
**Pull request by switzerrrland**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -113,6 +113,24 @@
...
+          <groupId>org.projectlombok</groupId>
+          <artifactId>lombok</artifactId>
+          <version>1.18.20</version>
+          <scope>compile</scope>
+      </dependency>
+    <dependency>
+      <groupId>com.epam.jdi</groupId>
+      <artifactId>jdi-light-html</artifactId>
+      <version>1.3.11</version>
+    </dependency>
```
Comment (shoggy) [inline pom.xml]: `jdi-light` and `jdi-light-html` have different version and can bring unexpected issues

File: src/main/java/ru/training/at/hw6/JdiHomePage.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+    private JdiLoginForm jdiLoginForm;
+
+    public void login(User user) {
+        userIcon.click();
+        jdiLoginForm.login(user);
+    }
+
+    public void selectMetalsAndColorsPageFromHeaderMenu() {
+        headerMenu.select(4);
```
Comment (shoggy) [inline src/main/java/ru/training/at/hw6/JdiHomePage.java]: An index is not the best option for String defined elements

**Review by shoggy — APPROVED** [review]

File: src/main/java/ru/training/at/hw6/JdiHomePage.java (hunk 2 above)
Comment (switzerrrland) [reply src/main/java/ru/training/at/hw6/JdiHomePage.java]: changed to string value

File: pom.xml (hunk 1 above)
Comment (switzerrrland) [reply pom.xml]: set both to the same version 1.3.11

**Pull request merged**